packages/site-kit/src/lib/components/Shell.svelte

```diff
@@ -30,6 +30,8 @@ The main shell of the application. It provides a slot for the top navigation, th
 	<slot name="top-nav" />
 {/if}
 
+<slot name="nav" />
```

Comment:
What's this needed for? There's the `top-nav` slot above already, is that one not enough?

Reply:
Oh ur right 😅. Removed!